### PR TITLE
build:utilities

### PR DESCRIPTION
**Pull request by Sakneb**

### CODE REVIEW

File: components/utilities/utilities.module.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@import "../../styles/variables";
+
+.utilities {
+  color: $light;
+
```
Comment (wwJavid) [inline components/utilities/utilities.module.scss:5]: .

File: components/utilities/utilities.module.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0px;
+  gap: 16px;
+
+  &_button {
+    color: $light;
+    background-color: $purple;
+    border-radius: 10%;
```
Comment (wwJavid) [inline components/utilities/utilities.module.scss:16]: why percentage?

File: components/utilities/utilities.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
+import styles from "./utilities.module.scss";
+
+function Utilities() {
+  return (
+    <div className={styles.utilities}>
+      <button className={styles.utilities_button}>Bizə qoşul</button>
```
Comment (wwJavid) [inline components/utilities/utilities.tsx:7]: Please, take the static words out of the code context: create object describing its usage, and put semantic words as key value pair.

**Review by wwJavid — CHANGES_REQUESTED** [review]
Please, resolve the comments.

**Comment by wwJavid** [comment]
> Please, resolve the comments.

Is this current view for you as well or it's related to my machine?

Screenshot 2023-05-26 at 19 10 07

File: components/utilities/utilities.tsx (hunk 3 above)
Comment (Sakneb) [reply components/utilities/utilities.tsx:7]: I couldn't fix only this

**Pull request merged**